prow/job/controller.py

```diff
@@ -867,18 +866,12 @@ def update_retried_job_run(self, build, job_name, current_job_id, new_job_id):
                 logger.error(f"cannot find job name {job_name} in {file_path}")
 
 
-def validate_required_info(release=None):
-    if os.environ.get(SYS_ENV_VAR_API_TOKEN) is None:
+def validate_required_info(required_env_vars):
+    missing_env_vars = [
+        var for var in required_env_vars if os.environ.get(var) is None]
+    if missing_env_vars:
         raise SystemExit(
-            f"Cannot find environment variable {SYS_ENV_VAR_API_TOKEN}")
-    if os.environ.get(SYS_ENV_VAR_GITHUB_TOKEN) is None:
-        raise SystemExit(
-            f"Cannot find environment variable {SYS_ENV_VAR_GITHUB_TOKEN}")
-    if os.environ.get(SYS_ENV_VAR_GCS_CRED_FILE) is None:
-        raise SystemExit(
-            f"Cannot find environment variable {SYS_ENV_VAR_GCS_CRED_FILE}")
-    if release and release not in VALID_RELEASES:
```

Comment:
Do we want to skip this check? Maybe we can check if the release prefix to avoid running commands other release versions like 4.18, 4.19.

Reply:
Hmm but this is just matter of promote command ... you can ignore this.